core/src/main/scala/ai/h2o/sparkling/backend/SharedBackendConf.scala

```diff
@@ -171,6 +171,8 @@ trait SharedBackendConf {
 
   def icedDir: Option[String] = sparkConf.getOption(PROP_ICED_DIR._1)
 
+  def sessionTimeout: Int = sparkConf.getInt(PROP_SESSION_TIMEOUT._1, PROP_SESSION_TIMEOUT._2)
```

Comment:
I'm wondering whether it isn't better to choose a different name for the property. IMHO, people will associate this name with `SparkSession` or `H2OContext`. WDYT?

Reply:
Good point, I didn't think about that. Maybe "RestApiTimeout" ?